code/script.js

```diff
@@ -0,0 +1,94 @@
+const USER = "IdaAspen";
+const REPOS_URL = `https://api.github.com/users/${USER}/repos`;
+const USER_INFO_URL = `https://api.github.com/users/${USER}`;
+
+const projectsContainer = document.getElementById("projects-container");
+const userContainer = document.getElementById("user-container");
+
+//get username and profile picture
+const getUserInfo = () => {
+  fetch(USER_INFO_URL)
+    .then((response) => response.json())
+    .then((data) => {
+      userContainer.innerHTML = `
+      <div class="user-card">
+      <img src="https://avatars.githubusercontent.com/u/80949028?v=4"
+       alt="Profile picture"><h2>User name: <a href="https://github.com/IdaAspen">${data.name}</a></h2>
+       <p>${data.bio}</p>
+       </div>
+      `;
+    });
+};
+
+/////////////////////// GET REPOS /////////////////////////////////////////
+//Get all repos w/ details
+const getRepos = () => {
+  fetch(REPOS_URL)
+    .then((response) => response.json())
+    .then((data) => {
+      let forkedRepos = data.filter(
+        (repo) => repo.fork && repo.name.startsWith("project-")
+      );
+      //Sorts filtered repos after dates
+      forkedRepos.sort(
+        (a, b) => new Date(b.created_at) - new Date(a.created_at)
+      );
+      forkedRepos = forkedRepos.reverse();
+
+      forkedRepos.forEach(
+        (repo) =>
+          (projectsContainer.innerHTML += `
+          <div class="project-card">
+          <h3><a href=${repo.html_url} target="_blank">${repo.name.replace(
+            "-",
+            " "
+          )}</a></h3>
+          <p>Most recent push: ${repo.pushed_at.substring(0, 10)}
```

Comment:
Interesting alternative, I hadn't considered. Could be quite useful when fetching other types of strings. (In my case, I used `${new Date(repo.pushed_at).toDateString()}`, but that one can be limiting because it only applies to dates).

Reply:
Yes, I tried both solutions and decided for this one